src/Stan/Report/Css.hs

```diff
@@ -0,0 +1,28 @@
+module Stan.Report.Css
+    ( stanCss
+    ) where
+
+import Clay (Css, (?))
+
+import qualified Clay as Css
+
+
+stanCss :: Css
+stanCss = do
+    Css.body ? do
+        Css.margin zero zero zero zero
+        Css.fontFamily [] [Css.sansSerif]
+        Css.color darkGrey
```

Comment:
I think, we can avoid using `qualified` import for `Clay` here. Will probably make writing CSS a bit easier